com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -221,6 +221,12 @@ void SetPerCameraShaderVariables(CommandBuffer cmd, ref CameraData cameraData)
 
             cmd.SetGlobalVector(ShaderPropertyId.screenSize, new Vector4(cameraWidth, cameraHeight, 1.0f / cameraWidth, 1.0f / cameraHeight));
 
+            // Calculate a bias value which corrects the mip lod selection logic when image scaling is active.
+            // We clamp this value to 0.0 or less to make sure we don't end up reducing image detail in the downsampling case.
+            float mipBias = Math.Min((float)-Math.Log(cameraWidth / scaledCameraWidth, 2.0f), 0.0f);
+            cmd.SetGlobalFloat(ShaderPropertyId.globalMipBias, mipBias);
+            cmd.SetGlobalFloat(ShaderPropertyId.globalMipBiasPow2, Mathf.Pow(2.0f, mipBias));
+
```

Comment:
are we still landing the mipBias PR as a separate PR? Asking since these changes are the same

Reply:
Yep, that commit was pulled into this PR temporarily so it'd represent final-like quality. It'll be removed when this becomes non-draft.